contrib/lib/src/databases.rs

```diff
@@ -320,12 +316,42 @@
 //! }
 //!
 //! #[get("/")]
-//! fn my_handler(conn: MyDatabase) -> Data {
-//!     load_from_db(&*conn)
+//! async fn my_handler(mut conn: MyDatabase) -> Data {
+//!     conn.run(|c| load_from_db(c)).await
 //! }
 //! # }
 //! ```
 //!
+//! // `run()` serializes access to the connection. To create an independent
```

Comment:
why is this commented out instead of deleted?
or is this just a reminder that the documentation here is wrong and needs fixing?

Reply:
Right; I commented out to indicate this doc was no longer valid.